### DISCUSSION TITLE
Reviewer

### DISCUSSION THREAD

**Discussion by pizoli** [SzFMV2021-Osz/warmup#12] [General]
Üdv @all!

Kérnék vallakit, aki review-ozza a kódomat.
Nem ismerek senkit még a csapatból...

Én is viszont megnézem az valakiét.

Köszönettel
Z

@SzFMV2021-Osz/everyone

**Comment by pintergreg** [comment]
@pizoli @SzFMV2021-Osz/everyone Ha hozzáadod az Everyone csoportot, akkor sorsol belőle valakit a GH. De ez a poszt is egy teljesen működő megoldás így.

**Comment by IKGSUI** [comment]
Hello @SzFMV2021-Osz/everyone !

Valakit megkérhetek, hogy reviewozza a kódomat?

Köszönöm,
Domink
